src/choreo/voter/fd_voter.c

```diff
@@ -68,14 +66,11 @@ fd_vote_txn_generate( fd_voter_t *                     voter,
                                  vote_instr_buf,
                                  vote_instr_size );
 
-    fd_sha512_t sha;
     fd_txn_t * txn_meta = (fd_txn_t *)fd_type_pun( out_txn_meta_buf );
-    fd_ed25519_sign( /* sig */ out_txn_buf + txn_meta->signature_off,
-                     /* msg */ out_txn_buf + txn_meta->message_off,
-                     /* sz  */ txn_size - txn_meta->message_off,
-                     /* public_key  */ node_pubkey->key,
-                     /* private_key */ voter->node_keypair,
-                     &sha );
+    voter->node_sign_fun( voter->voter_sign_arg,
```

Comment:
can you call this `validator_identity_sign_fun`? also can you make two sign args the same way?